### DISCUSSION TITLE
Inconclusive Result

### DISCUSSION THREAD

**Discussion by paulmccrady** [Azure/PSRule.Rules.Azure#2491] [Q&A]
What does this mean?

WARNING: Inconclusive result reported for 'myresource' @PSRule.Rules.Azure\Azure.ACR.Usage.

**Comment by BernieWhite** [comment]
@paulmccrady An inconclusive result means that the rule didn't pass or fail, because the rule didn't pass or fail.

In this case, I think this is a bug for the `Azure.ACR.Usage` rule so I'm going to log an issue for that. #2494

**Reply by paulmccrady** [reply]
ah ok, let me take a look at why the rule doesnt work against my environment then an advise.

**Reply by BernieWhite** [reply]
@paulmccrady Is this still an issue?

**Reply by paulmccrady** [reply]
@BernieWhite yes - will try and look at it next week to figure out why in my environment this occurs.

**Reply by paulmcclbg** [reply]
@BernieWhite we see odd behaviour with this rule. It reports all our ACRs as failure even though they are well under the 90% usage limit in the rule. Running an export against our environment and I do not see the listUsages resource. Is there a specific role / permission that is required to retrieve this data or is it due to an out of date api ? The omission of the sub resource is in itself weird as I would expect PSRule to report inconclusive result rather than a failure.

**Reply by BernieWhite** [reply]
Thanks for the info @paulmcclbg. Reading the listUsages should only be reader.

**Reply by paulmcclbg** [reply]
OK, but like I said, we do not get that sub resource returned in the json
